examples/README.md

```diff
@@ -31,8 +31,18 @@ ize tunnel up
 ize tunnel down
 ```
 
+### Connect to a container in the ECS
+```shell
+ize console goblin
+```
+
 ### Upload/Remove secrets
 ```shell
-ize secret set --file .infra/env/testnut/secrets/example-service.json --type ssm
-ize secret remove --type ssm --path /testnut/example-service
+ize secrets set --file .infra/env/testnut/secrets/example-service.json --type ssm
```

Comment:
From ize examples

secrets push
```
Examples:
  # Push secrets:
  
  # This will push secrets for "squibby" app
  ize secrets push squibby
  
  # This will push secrets for "squibby" app from a "example-service.json" file to the AWS SSM storage with force option (values will be overwritten if exist)
  ize secrets push squibby --backend ssm --file example-service.json --force
```